assets/quick-order-list.js

```diff
@@ -66,6 +66,7 @@ class QuickOrderList extends HTMLElement {
       add: 'ADD',
       update: 'UPDATE'
     }
+    this.mql = window.matchMedia('(min-width: 750px)');
```

Comment:
```suggestion
    this.mqlTablet = window.matchMedia('(min-width: 750px)');
```

Following what we do on the quantity-popover.js file